tests/neo4j/txfuncrun.py

```diff
@@ -24,6 +24,32 @@ def tearDown(self):
         self._driver.close()
         super().tearDown()
 
+    def test_simple_query(self):
+        def _test():
+            self._driver.close()
+            self._driver = get_driver(self._backend, userAgent="test")
+            self._session1 = self._driver.session("r", fetchSize=2)
+            tx = self._session1.beginTransaction()
```

Comment:
is this test in the right file? It's testing an explicit tx, while I thought this file was about transaction functions.

Reply:
I think it's been swapped with the test in `txrun.py` :)

Reply:
Oh yes. Well spotted, thanks!